Library/Formula/v8.rb

```diff
@@ -21,6 +21,7 @@ class V8 < Formula
 
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
   depends_on "readline" => :optional
+  depends_on "icu4c" => :recommended
```

Comment:
Idea is to serve close to the same v8 configuration that served on debian/ubuntu via libv8 and that Chrome have. Regardless `icu4c` distribution size, I guess there are not so many people who even require `v8`, and, for those who do that, distribution size doesn't actually matter.

I would like to insist to have ICU by default, but if from your perspective it should be optional, please confirm that.